extensions/tmtv/src/utils/hpViewports.ts

```diff
@@ -312,10 +311,12 @@ const fusionSAGITTAL = {
     },
     {
       options: {
-        colormap: 'hsv',
+        colormap: {
+          name: 'hsv',
+          opacityMapping: [[0.1, 0.9]],
```

Comment:
Is there an example of a non-double array instance?  Just seems odd to have a double array here.

Reply:
I'm not sure what is the best way to do this other than array of arrays, the idea is min value is mapped to 0 and max value is mapped to 1, and there is a linear interpolation for the rest. But if you provide this opacityMapping in the HP it can override that interpolation and add a point there which in this case it will add mapping of 0.1 to 0.9 opacity. Theoretically people can add as many points as they want. Other options is to do something like

```js
opacityMapping: [
 { value: 0.1, opacity: 0.9}
]
```

Reply:
I'd prefer an array of objects, as I find that easier to understand.

Reply:
I agree with Bill.